libs/watt/src/lib/components/stepper/watt-stepper.component.ts

```diff
@@ -60,9 +60,7 @@ export class WattStepperComponent extends MatStepper implements AfterViewInit {
   @Input() isCompleting = false;
 
   @ContentChildren(WattStepperStepComponent, { descendants: true })
-  override _steps!: QueryList<WattStepperStepComponent>;
-
-  override readonly steps!: QueryList<WattStepperStepComponent>;
+  declare _steps: QueryList<WattStepperStepComponent>;
```

Comment:
nitpick: TIL that this can be used too. Don't think I've seen it in this context 🤔

Reply:
If i dont do this, storybook will not build.. I seems not to have any side effects

Reply:
Or i could be wrong because now apps dont build..

Reply:
Ah used the wrong collection